seeding/schema.sql

```diff
@@ -0,0 +1,146 @@
+(45, 17.94979, -94.91386, 'Acayucan', 'MX');
+
+INSERT INTO alpha.botanist (botanist_id, botanist_name, botanist_email, botanist_phone_no) VALUES
+(1, 'Carl Linnaeus', 'carl.linnaeus@lnhm.co.uk', '(146)994-1635x35992'),
+(2, 'Gertrude Jekyll', 'gertrude.jekyll@lnhm.co.uk', '001-481-273-3691x127'),
+(3, 'Eliza Andrews', 'eliza.andrews@lnhm.co.uk', '(846)669-6651x75948');
+
+INSERT INTO alpha.plant (plant_id, plant_name, scientific_name, location_id, botanist_id) VALUES
+(0, 'Golden Pothos', 'Epipremnum Aureum', 1, 1),
+(1, 'Venus Flytrap', 'Dionaea Muscipula', 2, 2),
+(2, 'Corpse Flower', 'Amorphophallus Barteri', 3, 1),
+(3, 'Corpse Flower', 'Rafflesia Arnoldii', 1, 3),
+(4, 'Black Bat Flower', 'Tacca Chantrieri', 4, 1),
+(5, 'Pitcher Plant', 'Nepenthes Khasiana', 5, 1),
+(6, 'Wollemi Pine', 'Wollemia Nobilis', 6, 3),
+(8, 'False Bird of Paradise', 'Heliconia Schiedeana', 7, 3),
+(9, 'Cactus', 'Cactaceae', 8, 2),
+(10, 'Dragon Tree', 'Dracaena Draco', 9, 2),
+(11, 'Mexican Butterfly Weed', 'Asclepias Curassavica', 10, 2),
+(12, 'Angels Trumpet', 'Brugmansia X Candida', 11, 3),
+(13, 'Canna', 'Canna Striata', 12, 3),
+(14, 'Taro', 'Colocasia Esculenta', 13, 2),
+(15, 'Firecracker Bush', 'Cuphea David Verity', 14, 2),
+(16, 'Smoke Tree Spurge', 'Euphorbia Cotinifolia', 15, 2),
+(17, 'Sweet Potato', 'Ipomoea Batatas', 16, 1),
+(18, 'Variegated Tapioca', 'Manihot Esculenta Variegata', 17, 1),
+(19, 'Hardy Banana', 'Musa Basjoo', 18, 2),
+(20, 'Scarlet Sage', 'Salvia Splendens', 19, 1),
+(21, 'Flamingo Flower', 'Anthurium', 20, 3),
+(22, 'Bird of Paradise', 'Strelitzia Reginae', 21, 2),
+(23, 'Ti Plant', 'Cordyline Fruticosa', 22, 3),
+(24, 'Fig Tree', 'Ficus Carica', 23, 1),
+(25, 'Palm Tree', 'Arecaceae', 24, 2),
+(26, 'Dumb Cane', 'Dieffenbachia Seguine', 25, 1),
+(27, 'Peace Lily', 'Spathiphyllum', 26, 1),
+(28, 'Croton', 'Codiaeum variegatum', 27, 1),
+(29, 'Aloe Vera', 'Aloe Barbadensis Miller', 28, 2),
+(30, 'Rubber Plant', 'Ficus Elastica', 29, 1),
+(31, 'Snake Plant', 'Sansevieria Trifasciata', 30, 2),
+(32, 'Heartleaf Philodendron', 'Philodendron Hederaceum', 31, 2),
+(33, 'Umbrella Plant', 'Schefflera Arboricola', 32, 1),
+(34, 'Chinese Evergreen', 'Aglaonema Commutatum', 19, 2),
+(35, 'Swiss Cheese Plant', 'Monstera Deliciosa', 33, 1),
+(36, 'Bat Flower', 'Tacca Integrifolia', 34, 2),
+(37, 'Butterfly Orchid', 'Psychopsis Papilio', 35, 3),
+(38, 'African Violet', 'Saintpaulia Ionantha', 36, 2),
+(39, 'Blanket Flower', 'Gaillardia', 37, 1),
+(40, 'Amaryllis', 'Hippeastrum', 38, 3),
+(41, 'Angel Wings', 'Caladium Bicolor', 39, 1),
+(42, 'Spider Plant', 'Chlorophytum Comosum', 40, 1),
+(44, 'Norfolk Island Pine', 'Araucaria Heterophylla', 41, 2),
+(45, 'Wax Begonia', 'Begonia Semperflorens', 2, 2),
+(46, 'Pink Lantern', 'Medinilla Magnifica', 42, 3),
+(47, 'Red Powder Puff', 'Calliandra Haematocephala', 43, 3),
+(48, 'ZZ Plant', 'Zamioculcas Zamiifolia', 44, 1),
+(49, 'Jade Plant', 'Crassula Ovata', 45, 3),
```

Comment:
Never doubt a man on a mission, background music, and the Internet, apparently. Great to have this, as we can account for edge cases where certain entries don't contain a plant's common name,  only their scientific name; and vice versa. Will save us a lot of headache later if we ever have to retrace insertions. Nice for completeness too.